### DISCUSSION TITLE
size of activation

### DISCUSSION THREAD

**Discussion by evernerve** [LabeoTech/Umit#58] [General]
Hi
First of all i want to thank you for your efforts to improve this valuable toolbox.
i wanted to know if its possible to measure the size of a specific area in the brain that responds to visual stimuli for example v1 or other area with umit if its possible What will be the pipeline that i need?
best regards

**Comment by BrunoOFSouza** [comment]
Hi  @evernerve,

It is possible to measure the area of a brain region activated by a stimulus. In brief, you can load the brain activation map (fig 1) to the ***ROImanager*** app  and create an ROI corresponding to the activated area. You can use the option **Create >> From threshold...** to use a threshold value to delimit the ROI (figs 2-3). 
To get the area in mm<sup>2</sup>, you will need the pixel ratio of the recording in pixels per mm. In the ***ROImanager***, go to **Image >> Set pixel size** and enter the pixel value. Otherwise the app will give the area in pixel<sup>2</sup>.
The *area* value will be displayed in the table (fig 4). 
1. Activation map:
![actv_map](https://user-images.githubusercontent.com/72028622/231788010-b22440ca-6e3c-44e2-a336-bd8699ae248a.png)
2. Create the ROI:
![thr_roi_creation](https://user-images.githubusercontent.com/72028622/231788229-ce4d1264-5a94-4deb-b4ce-3a6e48606b13.png)
3. ROI created!
![actv_map_w_roi](https://user-images.githubusercontent.com/72028622/231788502-7e05900a-1240-4e7d-ad26-3e0ec88b8d28.png)
4. Get the ROI area:
![roi_table](https://user-images.githubusercontent.com/72028622/231786453-bf394393-75a3-4a47-ad98-8b8f8aa759f0.png)

**Reply by DevilsAdvocate2244** [reply]
thanks a lot but for pixel ratio What would be the suitable number?

**Reply by BrunoOFSouza** [reply]
Well, the pixel ratio will give the number of pixels per millimeters in order to be able to estimate the real size of the image. This can vary depending on the zoom (on the lenses) that you used during the recordings. If you always used the same parameters, what you can do is to take a snapshot of an object with known dimensions (e.g. a ruler) and measure the number of pixels per distance.

**Comment by DevilsAdvocate2244** [comment]
okay thank you Bruno.
khodafez
________________________________

**Comment by DevilsAdvocate2244** [comment]
Another question for threshold level what does it represent?
________________________________

**Reply by BrunoOFSouza** [reply]
The threshold represents the cut-off value (in pixel value if not normalized or 0-1 if normalized) that will be used to create the ROI. 
In the example above, the normalized threshold  of 0.4 means that the ROI encompasses an area where all pixel values are below 40% of all pixel values in the image. 

There are several ways to determine a threshold. Some more biased than others. The current option is a manual threshold. 
I am currently working on the *Auto Set* button to set the threshold automatically based on the image statistics. But this will only  be available on the next version of the toolbox.

For a description of the app's interface elements, check the app`s documentation [here](https://labeotech.github.io/Umit/documentation/userDocs/ROImanager.html#create-rois-using-threshold).

**Reply by DevilsAdvocate2244** [reply]
Yes it will be a great help. I think it will be okay after your guidance.
